audio/faust-devel/Portfile

```diff
@@ -38,7 +39,8 @@ if {${os.platform} eq "darwin" && ${os.major} > 14} {
 set llvm_prefix         ${prefix}/libexec/llvm-${llvm_version}
 build.env               PATH=${llvm_prefix}/bin:$env(PATH)
 
-depends_build           port:pkgconfig
+depends_build           port:pkgconfig \
+                        port:cmake
```

Comment:
Another nitpick. We prefer the dependencies to be sorted.

Reply:
We also have a development version of cmake available in the cmake-devel port. To allow the user to choose which of these they want to use, the dependency should instead be written `path:bin/cmake:cmake`.

Reply:
Fixed